### PR TITLE
Add test with multiple ops

### PR DESCRIPTION
**Pull request by priyaramani**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #73888

Differential Revision: [D34702376](https://our.internmc.facebook.com/intern/diff/D34702376)

### CODE REVIEW

**Comment by priyaramani** [comment]
@priyaramani has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34702376).

File: test/mobile/lightweight_dispatch/test_codegen_unboxing.cpp [hunk 1]
```diff
@@ -190,6 +191,30 @@ TEST(LiteInterpreterTest, DivideTensor) {
   AT_ASSERT(result_1.toList().get(0).toTensor().equal(expected_1));
   AT_ASSERT(result_1.toList().get(1).toTensor().equal(expected_2));
 }
+
+TEST(LiteInterpreterTest, MultipleOps) {
+  // Load check in model: basic.ptl
```
Comment (larryliu0820) [inline test/mobile/lightweight_dispatch/test_codegen_unboxing.cpp:196]: Maybe change this?

File: test/mobile/lightweight_dispatch/test_codegen_unboxing.cpp [hunk 2]
```diff
@@ -190,6 +191,30 @@ TEST(LiteInterpreterTest, DivideTensor) {
...
+  auto testModelFile = "multiple_ops.ptl";
+
+  // class Model(torch.nn.Module):
+  //           def __init__(self):
+  //               super(Model, self).__init__()
+  //               self.ops = torch.nn.Sequential(
+  //                   torch.nn.ReLU(),
+  //                   torch.nn.AdaptiveAvgPool2d(output_size=(1, 1)),
```
Comment (larryliu0820) [inline test/mobile/lightweight_dispatch/test_codegen_unboxing.cpp:204]: Also we don't have this op yet, right?

**Review by larryliu0820 — APPROVED** [review]
Looks good to me! Please address the comments before land.

**Pull request closed without merging**